--- canonical-topology/Poincare/Topology/Homology/TriangleFilling.lean
+++ DifferentialGeometry/External/CanonicalTopology/Topology/Homology/TriangleFilling.lean
@@ -31 +31 @@
-  have hf (i : Fin 3) (t : stdSimplex ℝ (Fin 2)) :
+  have hf (i : Fin 3) (t : Convexity.StdSimplex ℝ (Fin 2)) :
@@ -33 +33 @@
-        triangleBoundaryPaths p q r (i, stdSimplexHomeomorphUnitInterval t) := by
+        triangleBoundaryPaths p q r (i, Convexity.StdSimplex.homeomorphI t) := by
@@ -38 +38 @@
-        (stdSimplex.map i.succAbove t) = _
+        (Convexity.StdSimplex.map i.succAbove t) = _
@@ -40,3 +40,3 @@
-    change F (simplexTriangleHomeomorph (stdSimplex.map i.succAbove t)) = _
-    let z := planeTriangleEdge i (stdSimplexHomeomorphUnitInterval t)
-    have he : simplexTriangleHomeomorph (stdSimplex.map i.succAbove t) =
+    change F (simplexTriangleHomeomorph (Convexity.StdSimplex.map i.succAbove t)) = _
+    let z := planeTriangleEdge i (Convexity.StdSimplex.homeomorphI t)
+    have he : simplexTriangleHomeomorph (Convexity.StdSimplex.map i.succAbove t) =
@@ -46 +46 @@
-    exact (hF z).trans (triangleBoundaryMap_edge p q r i (stdSimplexHomeomorphUnitInterval t))
+    exact (hF z).trans (triangleBoundaryMap_edge p q r i (Convexity.StdSimplex.homeomorphI t))
