--- canonical-topology/Poincare/Topology/Homology/LocalOrientation.lean
+++ DifferentialGeometry/External/CanonicalTopology/Topology/Homology/LocalOrientation.lean
@@ -317 +317 @@
-    · rw [if_pos ha, if_pos hb, sign_pos (hpos (hb.trans ha.symm))]
+    · rw [ite_eq_left ha, ite_eq_left hb, sign_pos (hpos (hb.trans ha.symm))]
@@ -321 +321 @@
-      rw [if_pos ha, if_neg hb, sign_neg (hneg hba)]
+      rw [ite_eq_left ha, ite_eq_right hb, sign_neg (hneg hba)]
@@ -327 +327 @@
-      rw [if_neg ha, if_pos hb, sign_neg (hneg hba)]
+      rw [ite_eq_right ha, ite_eq_left hb, sign_neg (hneg hba)]
@@ -330 +330 @@
-      rw [if_neg ha, if_neg hb, sign_pos (hpos (hb'.trans ha'.symm))]
+      rw [ite_eq_right ha, ite_eq_right hb, sign_pos (hpos (hb'.trans ha'.symm))]
