--- before
+++ after
@@ -34 +34 @@
-  (exists_integralPathTriangle_boundary (PathConnectedSpace.somePath a _)
+  (exists_integralPathTriangle (PathConnectedSpace.somePath a _)
@@ -45,4 +45,15 @@
-  have h := (exists_integralPathTriangle_boundary (PathConnectedSpace.somePath a _)
-    (integralSimplexPath σ) (PathConnectedSpace.somePath a _)).choose_spec
-  simpa only [integralSingularConeTriangle, integralPathChain_simplexPath σ,
-    integralSingularConeZero_simplex] using h
+  have h (i : Fin 3) :
+      (TopCat.toSSet.obj (TopCat.of X)).δ i (integralSingularConeTriangle a σ) =
+        ![integralPathSimplex (integralSimplexPath σ),
+          integralPathSimplex (PathConnectedSpace.somePath a
+            (TopCat.toSSetObj₀Equiv ((TopCat.toSSet.obj (TopCat.of X)).δ 0 σ))),
+          integralPathSimplex (PathConnectedSpace.somePath a
+            (TopCat.toSSetObj₀Equiv ((TopCat.toSSet.obj (TopCat.of X)).δ 1 σ)))] i :=
+    (exists_integralPathTriangle (PathConnectedSpace.somePath a _)
+      (integralSimplexPath σ) (PathConnectedSpace.somePath a _)).choose_spec i
+  rw [integralSimplexChain_boundary_two, h 0, h 1, h 2]
+  change integralPathChain (integralSimplexPath σ) -
+      integralPathChain (PathConnectedSpace.somePath a _) +
+      integralPathChain (PathConnectedSpace.somePath a _) = _
+  rw [integralPathChain_simplexPath, integralSingularConeZero_simplex,
+    integralSingularConeZero_simplex]
