--- before
+++ after
@@ -6 +6 @@
-import DifferentialGeometry.Topology.Homology.ModuleHomologyClasses
+import DifferentialGeometry.External.CanonicalTopology.Topology.Homology.ModuleHomologyClasses
@@ -113 +113 @@
-    (K : Geometry.SimplicialComplex ℝ E) [Finite K.faces]
+    (K : Geometry.SimplicialComplex ℝ E)
@@ -124 +124 @@
-    (K : Geometry.SimplicialComplex ℝ E) [Finite K.faces]
+    (K : Geometry.SimplicialComplex ℝ E)
@@ -130 +130 @@
-    (K : Geometry.SimplicialComplex ℝ E) [Finite K.faces]
+    (K : Geometry.SimplicialComplex ℝ E)
@@ -137 +137 @@
-    (K : Geometry.SimplicialComplex ℝ E) [Finite K.faces]
+    (K : Geometry.SimplicialComplex ℝ E)
