--- before
+++ after
@@ -2 +2 @@
-import DifferentialGeometry.Topology.Homology.Homotopy
+import DifferentialGeometry.External.CanonicalTopology.Topology.Homology.Homotopy
@@ -130,2 +130 @@
-theorem sphereHurewiczTwoCanonical_iff_of_homeomorph [SimplyConnectedSpace X]
-    [SimplyConnectedSpace Y] (e : X ≃ₜ Y) :
+theorem sphereHurewiczTwoCanonical_iff_of_homeomorph (e : X ≃ₜ Y) :
@@ -139,2 +138 @@
-theorem sphereHurewiczThreeCanonical_iff_of_homeomorph [SimplyConnectedSpace X]
-    [SimplyConnectedSpace Y] (e : X ≃ₜ Y) :
+theorem sphereHurewiczThreeCanonical_iff_of_homeomorph (e : X ≃ₜ Y) :
@@ -157 +155 @@
-theorem hurewiczTwoBijective_iff_of_homeomorph [SimplyConnectedSpace X] [SimplyConnectedSpace Y]
+theorem hurewiczTwoBijective_iff_of_homeomorph
@@ -167 +165 @@
-theorem hurewiczThreeBijective_iff_of_homeomorph [SimplyConnectedSpace X] [SimplyConnectedSpace Y]
+theorem hurewiczThreeBijective_iff_of_homeomorph
@@ -209,2 +207 @@
-theorem hurewiczLowDegreeFrontier_iff_of_homeomorph [SimplyConnectedSpace X]
-    [SimplyConnectedSpace Y] (e : X ≃ₜ Y) :
+theorem hurewiczLowDegreeFrontier_iff_of_homeomorph (e : X ≃ₜ Y) :
