--- before
+++ after
@@ -1,2 +1,2 @@
-import DifferentialGeometry.Topology.Homology.EuclideanLocalVanishing
-import DifferentialGeometry.Topology.Homology.Homotopy
+import DifferentialGeometry.External.CanonicalTopology.Topology.Homology.EuclideanLocalVanishing
+import DifferentialGeometry.External.CanonicalTopology.Topology.Homology.Homotopy
@@ -58 +58 @@
-    [IsManifold (𝓡 3) ∞ M] [T2Space M] [CompactSpace M] [SimplyConnectedSpace M] (x : M)
+    [T2Space M] [CompactSpace M] [SimplyConnectedSpace M] (x : M)
@@ -68,2 +68,2 @@
-    {M : Type u} [TopologicalSpace M] [ChartedSpace (EuclideanSpace ℝ (Fin 3)) M]
-    [IsManifold (𝓡 3) ∞ M] [T2Space M] [CompactSpace M] [SimplyConnectedSpace M] (x : M) :
+    {M : Type u} [TopologicalSpace M]
+    [T2Space M] [CompactSpace M] [SimplyConnectedSpace M] (x : M) :
