--- before
+++ after
@@ -1,3 +1,3 @@
-import DifferentialGeometry.Topology.Homology.Cochains
-import DifferentialGeometry.Topology.Homology.PathChains
-import DifferentialGeometry.Topology.Homology.SimplexBoundary
+import DifferentialGeometry.External.CanonicalTopology.Topology.Homology.Cochains
+import DifferentialGeometry.External.CanonicalTopology.Topology.Homology.PathChains
+import DifferentialGeometry.External.CanonicalTopology.Topology.Homology.SimplexBoundary
