--- before
+++ after
@@ -1 +1 @@
-import DifferentialGeometry.Topology.Homology.CompactlySupportedCohomology.CapNaturality
+import DifferentialGeometry.External.CanonicalTopology.Topology.Homology.CompactlySupportedCohomology.CapNaturality
@@ -8,2 +8,2 @@
-import DifferentialGeometry.Topology.Homology.TotallyDisconnectedZero
-import DifferentialGeometry.Topology.Homology.Homotopy
+import DifferentialGeometry.External.CanonicalTopology.Topology.Homology.TotallyDisconnectedZero
+import DifferentialGeometry.External.CanonicalTopology.Topology.Homology.Homotopy
