--- before
+++ after
@@ -1,3 +1,3 @@
-import DifferentialGeometry.Topology.Homology.RelativeFunctoriality
-import DifferentialGeometry.Topology.Homology.Integral
-import DifferentialGeometry.Topology.Homology.RelativeHomeomorphism
+import DifferentialGeometry.External.CanonicalTopology.Topology.Homology.RelativeFunctoriality
+import DifferentialGeometry.External.CanonicalTopology.Topology.Homology.Integral
+import DifferentialGeometry.External.CanonicalTopology.Topology.Homology.RelativeHomeomorphism
