--- before
+++ after
@@ -3 +2,0 @@
-
@@ -16 +15 @@
-    (hf : Topology.IsOpenEmbedding f) (K : Compacts X) :
+    (hf : _root_.Topology.IsOpenEmbedding f) (K : Compacts X) :
@@ -21 +20 @@
-    (hf : Topology.IsOpenEmbedding f) (K : Compacts X) :
+    (hf : _root_.Topology.IsOpenEmbedding f) (K : Compacts X) :
@@ -29 +28 @@
-    (hf : Topology.IsOpenEmbedding f) (K L : Compacts X) (h : K ≤ L)
+    (hf : _root_.Topology.IsOpenEmbedding f) (K L : Compacts X) (h : K ≤ L)
@@ -53 +52 @@
-    (hf : Topology.IsOpenEmbedding f) :
+    (hf : _root_.Topology.IsOpenEmbedding f) :
@@ -81 +80 @@
-    (n : ℕ) (f : ContinuousMap X Y) (hf : Topology.IsOpenEmbedding f) (K : Compacts X)
+    (n : ℕ) (f : ContinuousMap X Y) (hf : _root_.Topology.IsOpenEmbedding f) (K : Compacts X)
@@ -90 +89 @@
-    (n : ℕ) (f : ContinuousMap X Y) (hf : Topology.IsOpenEmbedding f) (K : Compacts X)
+    (n : ℕ) (f : ContinuousMap X Y) (hf : _root_.Topology.IsOpenEmbedding f) (K : Compacts X)
@@ -102 +101 @@
-    (n : ℕ) (f : ContinuousMap X Y) (hf : Topology.IsOpenEmbedding f)
+    (n : ℕ) (f : ContinuousMap X Y) (hf : _root_.Topology.IsOpenEmbedding f)
@@ -114 +113 @@
-    (n : ℕ) (f : ContinuousMap X Y) (hf : Topology.IsOpenEmbedding f) :
+    (n : ℕ) (f : ContinuousMap X Y) (hf : _root_.Topology.IsOpenEmbedding f) :
@@ -134 +133 @@
-      Topology.IsOpenEmbedding.id = LinearMap.id := by
+      _root_.Topology.IsOpenEmbedding.id = LinearMap.id := by
@@ -140 +139 @@
-    Topology.IsOpenEmbedding.id K K hmap (show MapsTo (ContinuousMap.id X)
+    _root_.Topology.IsOpenEmbedding.id K K hmap (show MapsTo (ContinuousMap.id X)
@@ -148 +147 @@
-    (hf : Topology.IsOpenEmbedding f) (hg : Topology.IsOpenEmbedding g) :
+    (hf : _root_.Topology.IsOpenEmbedding f) (hg : _root_.Topology.IsOpenEmbedding g) :
